e2e/playwright.config.js

```diff
@@ -21,7 +21,10 @@ export default defineConfig({
   // Opt out of parallel tests on CI.
   workers: process.env.CI ? 1 : undefined,
   // Use 'blob' for CI to allow merging of reports. See https://playwright.dev/docs/test-reporters
-  reporter: process.env.CI ? 'blob' : 'html',
+  reporter: process.env.CI
+  ? [['blob']]  :
+    // Never open the HTML report to prevent terminal from hanging on test failure
+    [['html', { open: 'never' }]],
```

Comment:
My impression is that we only want to do "open: never" when running in Docker. I don't think there's any issue with the report showing when running natively. That said, I could see an argument for wanting the two (native and container) to have consistent behavior. What are your thoughts?

Reply:
>wanting the two (native and container) to have consistent behavior
This is kind of what I was thinking - we've been trying to make the two consistent throughout the e2e work. Also recommend using `make e2e-show-report`.  Would propose this unless we strongly feel it should be set in the Dockerfile env instead just for container runs. 


Makes me wonder if then `blob` should also be `open:never` - if they were to set CI to true locally